packages/components/src/LegacyNewsCard/LegacyNewsCard.tsx

```diff
@@ -90,6 +115,17 @@ export const LegacyLoadingFeedCard = () => (
   </LegacyCard>
 );
 
+export const LegacyLoadingMobileCard = () => (
+  <LegacyMobileNewsCard display={['block', 'block', 'none']} px={3}>
```

Comment:
Reich mal das Display lieber aus der Komponente darüber durch, oder noch besser, leg einfach eine Box in der Komponente darüber drum, sodass es gar nicht Teil der Komponente ist.